lib/core/include/irods/msParam.h

```diff
@@ -94,7 +94,8 @@ typedef struct ValidKeyWd {
 
 #define RESC_NAME_FLAG          0x1
 #define DEST_RESC_NAME_FLAG     0x2
-// BACKUP_RESC_NAME has been deprecated.
+// BACKUP_RESC_NAME has been removed in iRODS 5.0.
+// The keyword is left so we do not reuse old values.
 #define BACKUP_RESC_NAME_FLAG   0x4
```

Comment:
This is supposed to be commented out as well.
```cpp
//#define BACKUP_RESC_NAME_FLAG   0x4 // Removed in iRODS 5.0.
```
The comment just serves as a reminder of what 0x4 originally mapped to and that we should not be re-purposing the value.

Reply:
Gotcha, will get RN.

Reply:
Done.